docs/services/frame-system/_index.md

```diff
@@ -115,11 +115,125 @@ The resulting tree of reference frames looks like:
-2. `FrameSystemConfig`: Returns a topologically sorted list of all the reference frames monitored by the frame system.
-Any [supplemental transforms](#supplemental-transforms) are also merged into the tree, topologically sorted, and returned.
+| Method Name | Description |
+| ----- | ----------- |
+| [`TransformPose`](#transformpose) | Transform a pose measured in one reference frame to the same pose as it would have been measured in another. |
+<!-- | [`FrameSystemConfig`](#framesystemconfig) | Return a topologically sorted list of all the reference frames monitored by the frame system. | -->
+
+<!--
+### FrameSystemConfig
+
+Returns a topologically sorted list of all the reference frames monitored by the frame system. Any [supplemental transforms](#supplemental-transforms) are also merged into the tree, sorted, and returned.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `additional_transforms` (Optional[List[[viam.proto.common.Transform](https://python.viam.dev/autoapi/viam/proto/common/index.html#viam.proto.common.Transform)]]): A list of [additional transforms](#supplemental-transforms).
+
+**Returns:**
+
+- `frame_system` (List[[viam.proto.robot.FrameSystemConfig](https://python.viam.dev/autoapi/viam/proto/robot/index.html#viam.proto.robot.FrameSystemConfig)]): The configuration of a given robot’s frame system.
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/robot/client/index.html#viam.robot.client.RobotClient.get_frame_system_config).
+
+```python {class="line-numbers linkable-line-numbers"}
+# Get a list of each of the reference frames configured on the robot. 
+frame_system = await robot.get_frame_system_config()
+print(f"Frame System Configuration: {frame_system}")
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- `ctx` [(`Context`)](https://pkg.go.dev/context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
+- `additionalTransforms` (Optional[[referenceframe.LinkInFrame](https://pkg.go.dev/go.viam.com/rdk@v0.2.36/referenceframe#LinkInFrame)]): A list of [additional transforms](#supplemental-transforms).
+
+**Returns:**
+
+- `error` [(`error`)](https://pkg.go.dev/builtin#error): An error, if one occurred.
+- `framesystemparts` [(`framesystemparts.Parts`)](https://pkg.go.dev/go.viam.com/rdk/spatialmath#Pose): The individual parts that make up a robot's frame system.
+
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/components/arm#Arm).
+
+```go {class="line-numbers linkable-line-numbers"}
+// Print the Frame System configuration
+frameSystem, err := robot.FrameSystemConfig(context.Background(), nil)
+fmt.Println(frameSystem)
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+-->
+
+### TransformPose
+
+Transform a given source Pose from the reference frame to a new specified destination which is a reference frame.
```

Comment:
@npentrel you are correct; this is not specific to arms.

The purpose of this call it to let the user easily do a bunch of complicated math based on where they set up their components to be located; no need for an arm specifically to be involved.

Example: if a 3d camera observes an XYZ point in space, and you want to know "where is that point relative to this gantry/board/whatever". Or if you were building a stereo camera system, and were comparing the poses of two different cameras to one another, especially if one or both are able to move.

One other example use case of this would indeed be to create poses which can be passed to the arm API. However this would not be needed for the motion API, since the motion service supports making this conversion on the backend.

Reply:
```suggestion
Transform a given source pose from the reference frame to a new specified destination reference frame.
For example, if a 3D camera observes a point in space you can use this method to calculate where that point is relative to another object.
```

^ Is this correct @biotinker ?